syntax = "proto3";

message UserRequest {
  int32 user_id = 1;
}

message UserResponse {
  int32 user_id = 1;
  string name = 2;
  string email = 3;
  string phone = 4;
}

service UserService {
  rpc GetUser (UserRequest) returns (UserResponse);
}

